test/rocarray/base.jl

```diff
@@ -27,6 +27,31 @@ end
     @test collect(c) == 3:4
 end
 
+@testset "resize!" begin
```

Comment:
Can you also add a check, for each of the 3 resize kinds, that `length(a_h) == length(a_d)`?

Reply:
Even though normal equality between abstract arrays will check for that, I will write a line to check for the length too.